### PR TITLE
remove feature new badges

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/DUDIl02I/6763-remove-badges-from-members-list-3

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Remove 'badges' from members list (3)](https://trello.com/c/DUDIl02I/6763-remove-badges-from-members-list-3)

File: orcid-web/src/main/resources/freemarker/includes/ng2_templates/member-details-ng2-template.ftl [hunk 1]
```diff
@@ -45,61 +45,13 @@
 	                <div  *ngIf="currentMemberDetails.integrations">
 		                <div *ngFor="let integration of currentMemberDetails.integrations">
 		                    <p><b>{{integration.name}}</b>&nbsp;<em>{{integration.stage}}</em></p>
-		                    <div *ngIf="!newBadgesEnabled && integration.badgeAwarded">
```
Comment (amontenegro) [inline orcid-web/src/main/resources/freemarker/includes/ng2_templates/member-details-ng2-template.ftl:48]: @DanielPalafox We dont want to remove the code right away, we want to hide it behind a togglz, so, please roll this code back

File: orcid-core/src/main/java/org/orcid/core/salesforce/dao/impl/SalesForceDaoImpl.java [hunk 2]
```diff
@@ -555,14 +555,8 @@ private String retrieveParentOrgNameFromSalesForce(String accessToken, String co
      * 
      */
     private List<Integration> retrieveIntegrationsFromSalesForce(String accessToken, String memberId) throws SalesForceUnauthorizedException {
-        String query = new String();
-        
-        if(Features.NEW_BADGES.isActive()) {
-            query = "SELECT Integration__c.Id, Integration__c.Name, Integration__c.Description__c, Integration__c.Integration_Stage__c, Integration__c.Level__c, Integration__c.BadgeAwarded__c, (Select Id, Name, Status__c, Badge__c from Achievements__r Where Status__c = 'Awarded') from Integration__c WHERE Integration__c.inactive__c=FALSE And Organization__c='%s'";
```
Comment (amontenegro) [inline orcid-core/src/main/java/org/orcid/core/salesforce/dao/impl/SalesForceDaoImpl.java:561]: @DanielPalafox please use this query on like 558, we want to keep this query and not the one in the `else` clause

Comment (DanielPalafox) [reply orcid-core/src/main/java/org/orcid/core/salesforce/dao/impl/SalesForceDaoImpl.java:561]: Ok

File: orcid-web/src/main/resources/freemarker/includes/ng2_templates/member-details-ng2-template.ftl (hunk 1 above)
Comment (DanielPalafox) [reply orcid-web/src/main/resources/freemarker/includes/ng2_templates/member-details-ng2-template.ftl:48]: Ok, I will roll back the code

**Pull request merged**